cunumeric/linalg/cholesky.py

```diff
@@ -35,9 +34,7 @@
 def transpose_copy_single(
     context: Context, input: Store, output: Store
 ) -> None:
-    task = cast(
-        ManualTask, context.create_task(CuNumericOpCode.TRANSPOSE_COPY_2D)
-    )
+    task = context.create_manual_task(CuNumericOpCode.TRANSPOSE_COPY_2D)
```

Comment:
@magnatelee question here, should this be a manual task? Previously it did not have `manual=True` but mypy will complain if it allowed to be an auto task:
```
cunumeric/linalg/cholesky.py: note: In function "cholesky":
cunumeric/linalg/cholesky.py:223:51: error: Argument 3 to "transpose_copy" has incompatible type "StorePartition"; expected "Store"  [arg-type]
        transpose_copy(context, Rect(hi=color_shape), p_input, p_output)
                                                      ^
cunumeric/linalg/cholesky.py:223:60: error: Argument 4 to "transpose_copy" has incompatible type "StorePartition"; expected "Store"  [arg-type]
```
It is store *partitions* that are definitely passed in around line 223

Reply:
it should be an auto task and not a manual task.

Reply:
https://github.com/nv-legate/cunumeric/pull/413/commits/875603c4977613cdf78b2ad60eaf4710d6c9d3bd  brain hiccup, mypy error was with default `create_task` and is fixed as expected with explicit `create_auto_task`